builder/oracle/bmcs/artifact.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2017 Oracle America, Inc.
+// The contents of this file are subject to the Mozilla Public License Version
+// 2.0 (the "License"); you may not use this file except in compliance with the
+// License. If a copy of the MPL was not distributed with this file, You can
+// obtain one at http://mozilla.org/MPL/2.0/
+
+package bmcs
+
+import (
+	"fmt"
+	client "github.com/mitchellh/packer/builder/oracle/bmcs/client"
```

Comment:
I believe the pattern is that SDKs used for accessing a platform are vendored out of the repo they originate from, rather than directly placed under the builders. This is the case for the AWS, Azure, etc SDKs. Including the entire client code under the builder would seem to break that pattern.

(That said, I couldn't find a repo in github for the Go implementation of the SDK, although Java and Python implementations are there.)

Note: opinions are my own as a Packer user and don't represent any official review.